<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>front_ADAS_setting</class>
 <widget class="QDialog" name="front_ADAS_setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/image/image/return.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="tittleButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>170</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>下一步</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(71, 154, 255)</string>
   </property>
   <property name="text">
    <string>前ADAS设置</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>301</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>31</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="high_low_Slider">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>30</y>
       <width>20</width>
       <height>101</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="highLabel">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>10</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 10pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>高</string>
     </property>
    </widget>
    <widget class="QLabel" name="lowLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>低</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="left_right_Slider">
     <property name="geometry">
      <rect>
       <x>29</x>
       <y>9</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="leftLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>8</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>左</string>
     </property>
    </widget>
    <widget class="QLabel" name="rightLabel">
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>8</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>右</string>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="HorizontalLine">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>200</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:
rgb(255, 8, 12)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="VerticelLine">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>20</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 2, 18)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
